install_config/configuring_local.adoc

```diff
@@ -27,37 +27,9 @@ The local volume provisioner allows using local storage within {product-title} a
 
 [NOTE]
 ====
-Local volumes is an alpha feature and may change in a future release of {product-title}.
+Local volumes is a technical preview feature and may change in a future release of {product-title}.
```

Comment:
s/technical preview/Technology Preview 

You should also add the IMPORTANT note: https://mojo.redhat.com/docs/DOC-1124368

Reply:
Replaced with IMPORTANT from the mojo page.